index.html

```diff
@@ -61,21 +65,17 @@
 							box-shadow: 0px 3px 2px rgba(0, 0, 0, 0.2);
 						}
 
-						paper-tabs::shadow #selectionBar {
+						paper-tabs #selectionBar {
 							background-color: #b12d2b;
 						}
-
-						paper-tab::shadow paper-ripple#ink {
-							color: #b12d2b;
-						}
 					</style>
-					<paper-tabs selected="js" class="js-app-tabs">
-						<paper-tab name="js">JavaScript</paper-tab>
-						<paper-tab name="ctojs">Compile-to-JS</paper-tab>
-						<paper-tab name="labs">Labs</paper-tab>
+					<paper-tabs selected="{{selected-tab}}" class="js-app-tabs">
```

Comment:
it's weird to use a js property name that "-" separated. `selectedTab` would be better.